migrate.py

```
@@ -127,55 +150,61 @@ def turn_off_vms(v_center: VCenter, servers: Servers):
+                print(f"Server '{server.host.name}' ({server.host.moid}) not found")
+                continue
+            if current_host.runtime.powerState == vim.HostSystem.PowerState.poweredOff:
+                print(f"Server '{server.host.name}' ({server.host.moid}) is already off")
+                continue
+
+            is_shutdown_plan = True
+            if server.destination:
+                dist_host = conn.get_host_system(server.destination.moid)
+                if dist_host:
+                    if dist_host.runtime.powerState == vim.HostSystem.PowerState.poweredOff:
+                        dist_ilo = Ilo(dist_host.ip, dist_host.user, dist_host.password)
```

Ruff (0.11.9)

Local variable `dist_ilo` is assigned to but never used

Remove assignment to unused variable `dist_ilo`

(F841)